README.md

```diff
@@ -1,12 +1,16 @@
 # Experimenting with grading workflows
 
-This repository is the wild wild west playground for experimenting with workflows
-that allow a course instructor to prepare material, distribute it to learners,
-and for them to then turn in their homework.
+This under development package supports several activities related to developing 
+online courses using Jupyter notebooks including:
+
+1. preparing course materials 
+2. distributing course materials to learners in a GitHub classroom environment
+3. automatic feedback on student assignments using CircleCi and
+4. autograding infrastructure for student assignments
 
 ## Try it out!
 
-Install with `pip install -e.` from the directory of this README.
+Install locally with `pip install -e.` from the directory of this README.
 
 Then run `nbauthor master/01-lecture.ipynb`. This will take the master notebook,
```

Comment:
@betatim may i update this? nbauthor runs differently now but just want to ensure i'm not missing something.